pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -62,6 +89,7 @@ stdenv.mkDerivation rec {
     homepage = "https://lammps.sandia.gov";
     license = licenses.gpl2Plus;
     platforms = platforms.linux;
-    maintainers = [ maintainers.costrouc ];
+    broken = (blas.isILP64 != lapack.isILP64);
```

Comment:
This only takes care that both are the same. However, the package most likely does not run if BLAS and LAPACK are provided as `ILP64` (usually segfaults at runtime). Packages that can handle both have need to have some kind of configure option to indicate to the build system which one should be used (see for example https://github.com/NixOS/nixpkgs/blob/d29e6a52554718691c8b5b0f16d68fad20c3110b/pkgs/applications/science/chemistry/nwchem/default.nix#L133).

Reply:
You are correct, I also experienced this when I added 64 bit support to octave.. Now it should be good.